.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### New Issue Checklist
+
+- [ ] I have tried with the latest version of OHHTTPStubs
+- [ ] I have read the [README](https://github.com/AliSoftware/OHHTTPStubs/blob/master/README.md)
+- [ ] I have read the [Using the right Swift Version of `OHHTTPStubs` for your project](https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-of-ohhttpstubs-for-your-project) section
+- [ ] I have searched in the [existing issues](https://github.com/AliSoftware/OHHTTPStubs/issues?utf8=✓&q=is%3Aissue)
+
+### Environment
+
+- version of OHHTTPStubs: [LIB VERSION HERE]
+- integration method you are using:
+  * [ ] Cocoapods
+  * [ ] Carthage
+  * [ ] submodule
+  * [ ] other
+- version of the tool you use: [INSERT VERSION HERE]
+
+### Issue Description
+
+[DESCRIBE YOUR ISSUE HERE]
+
+##### Complete output when you encounter the issue (if any)
+
+[INSERT OUTPUT HERE]
```

Comment:
We should insert 3-backticks before and after that line to encourage people to put their output in them and have a nicely-formatted terminal output

_We could even prepare a `<details>` block for them (so that if the terminal output they paste is long it doesn't make the issue description too unreadable), but I'm afraid doing that would be disturbing to people not knowing this tag_